RealmWeaver/ModuleWeaver.cs

```diff
@@ -810,6 +809,83 @@ private static bool IsRealmList(PropertyDefinition property)
         return property.PropertyType.Name == "RealmList`1" && property.PropertyType.Namespace == "Realms";
     }
 
+    private static bool IsDateTimeOffset(PropertyDefinition property)
```

Comment:
I know we've been using strings already, but would it be nicer to create something like
```
private static bool Is<T>(PropertyDefinition property)
{
    return property.PropertyType.Name == typeof(T).Name && ...
}
```
?